docs/install.md

```diff
@@ -0,0 +1,70 @@
+# Installing the Knative Pipelines component
+
+Before you can run Knative Pipelines, you must install the Knative Pipelines components
+in your Kubernetes cluster. Use this page to add the component to an existing 
+Kubernetes cluster.
+
+## Before you begin
+
+For complete installation instructions of Knative components, see [Installing Knative](https://github.com/knative/docs/blob/master/install/README.md).
+
+## Adding the Knative Pipelines component
+
+To add only the Knative Pipelines component to an existing installation:
+
+1. Run the
+   [`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply)
+   command to install [Knative Pipelines](https://github.com/knative/build-pipeline) and its
+   dependencies:
+   ```bash
+   kubectl apply --filename https://storage.googleapis.com/knative-releases/build-pipeline/latest/release.yaml
+   ```
+1. Run the
+   [`kubectl get`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#get)
+   command to monitor the Knative Pipelines components until all of the components
+   show a `STATUS` of `Running`:
+
+   ```bash
+   kubectl get pods --namespace knative-build-pipeline
+   ```
+
+   Tip: Instead of running the `kubectl get` command multiple times, you can
+   append the `--watch` flag to view the component's status updates in real
+   time. Use CTRL + C to exit watch mode.
+
+You are now ready to create and run Knative Pipelines: 
+ - See [Knative Pipeline tutorial](./tutorial.md) to get started. 
+ - Look at [the examples](https://github.com/knative/build-pipeline/tree/master/examples)
+
+
+## Configuring Knative Pipelines
+
+### How are resources shared between tasks?
```

Comment:
sorry for the late scope creep, but could you mention here also these configurable images: https://github.com/knative/build-pipeline/blob/master/config/controller.yaml#L30 ? ❤️ 😅 

![image](https://user-images.githubusercontent.com/432502/52749608-88ebbe00-2f9e-11e9-8037-9309bd4bc80d.png)

Reply:
Changing these images requires editing the release yaml by hand or creating a new custom `release.yaml`, which are advanced cases that I don't think it belong here.

Let me know if you want to add it anyways

Reply:
> can't say no to this cat :D

hahaha :D

> Changing these images requires editing the release yaml by hand or creating a new custom `release.yaml`, which are advanced cases that I don't think it belong here.

hm interesting! i have to say i do still feel like it's worth documenting - but as a side note i find it weird that we can configure 1 of these images using a configmap and the rest are at runtime. that's another story tho :)